modules/bitgo/test/v2/unit/tss/ecdsa.ts

```diff
@@ -71,61 +70,33 @@ describe('test tss helper functions', function () {
   describe('encryptNShare', function () {
     it('should encrypt n share', async function () {
       for (let i = 2; i <= 3; i++) {
-        const encryptedNShare = await ECDSAMethods.encryptNShare({
-          keyShare: userKeyShare,
-          recipientIndex: i,
-          senderGpgPrivateArmor: userGpgKeypair.privateKey,
-          recipientGpgPublicArmor: bitgoGpgKeypair.publicKey,
-        });
-        const decryptedMessage = await readSignedMessage(encryptedNShare.encryptedPrivateShare, userGpgKeypair.publicKey, bitgoGpgKeypair.privateKey);
-        decryptedMessage.should.equal(userKeyShare.nShares[i].u);
-        const publicKey = userKeyShare.pShare.y;
+        const encryptedNShare = await ECDSAMethods.encryptNShare(userKeyShare, i, bitgoGpgKeypair.publicKey, userGpgKeypair.privateKey);
+        const decryptedMessage = '0x' + await readSignedMessage(encryptedNShare.encryptedPrivateShare, userGpgKeypair.publicKey, bitgoGpgKeypair.privateKey);
+        decryptedMessage.should.equal(userKeyShare.nShares[i].u + userKeyShare.nShares[i].n.slice(2));
+        const publicKey = userKeyShare.pShare.y + userKeyShare.pShare.chaincode;
         encryptedNShare.i.should.equal(i);
         encryptedNShare.j.should.equal(1);
-        encryptedNShare.publicShare.should.equal(publicKey);
+        ('0x' + encryptedNShare.publicShare).should.equal(publicKey);
       }
     });
 
     it('should error for invalid recipient', async function () {
-      await encryptNShare({
-        keyShare: userKeyShare,
-        recipientIndex: 1,
-        senderGpgPrivateArmor: userGpgKeypair.privateKey,
-        recipientGpgPublicArmor: bitgoGpgKeypair.publicKey,
-      }).should.be.rejectedWith('Invalid recipient');
-      await encryptNShare({
-        keyShare: backupKeyShare,
-        recipientIndex: 2,
-        senderGpgPrivateArmor: userGpgKeypair.privateKey,
-        recipientGpgPublicArmor: bitgoGpgKeypair.publicKey,
-      }).should.be.rejectedWith('Invalid recipient');
-      await encryptNShare({
-        keyShare: bitgoKeyShare,
-        recipientIndex: 3,
-        senderGpgPrivateArmor: userGpgKeypair.privateKey,
-        recipientGpgPublicArmor: bitgoGpgKeypair.publicKey,
-      }).should.be.rejectedWith('Invalid recipient');
+      await encryptNShare(userKeyShare, 1, userGpgKeypair.privateKey, bitgoGpgKeypair.publicKey).should.be.rejectedWith('Invalid recipient');
```

Comment:
I think since we are only switching public <--> private keys in incorrect places as far as I understand, the error should be something like invalid private key / public key given for encryption right rather than invalid recipient?